roverrobotics_driver/launch/mini.launch

```diff
@@ -0,0 +1,43 @@
+<launch>
+    <arg name="driver_node_name" default="roverrobotics_driver" />
+    <arg name="config_locks" default="$(find roverrobotics_driver)/config/twist_mux_locks.yaml" />
+    <arg name="config_topics" default="$(find roverrobotics_driver)/config/twist_mux_topics_locks.yaml" />
+    <param name="device_port" value="can0" />
+    <param name="comm_type" value="can" />
+    <param name="robot_type" value="mini" />
+    <param name="robot_mode" value="TRACTION_CONTROL" />
```

Comment:
Let's use INDEPENDENT WHEEL as the default mode

Reply:
changed for Pro2 as well